.github/workflows/test.yml

```diff
@@ -9,10 +9,10 @@ on:
     - master
 
 jobs:
-  build:
+  python-django:
     runs-on: ubuntu-latest
     strategy:
-      max-parallel: 5
+      fail-fast: false
```

Comment:
This would be also not so great for the shared nature of the GitHub action runners in the Jazzband org, although I understand why it would be useful.

Reply:
Fine with me, undone. – Just for curiosity: Does the project have dedicated CI runners?

Reply:
No, it's shared with the other Jazzband projects.

Reply:
I intended "the Jazzband project". Does GitHub provide dedicated infrastructure for Free Software projects? 🤔